Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/OverlayMessagingService.java

```diff
@@ -135,15 +153,20 @@ private void setDataToViews(String bundle) {
             LinearLayout buttonLayout = messageView.findViewById(R.id.button_view);
             MaterialButton buttonOk = messageView.findViewById(R.id.button_ok);
             TextView buttonCancel = messageView.findViewById(R.id.button_cancel);
+            ImageView cancelButtonImage = messageView.findViewById(R.id.cancel_button_image);
+            TextView buttonCancelRight = messageView.findViewById(R.id.button_cancel_right);
             title.setText(data.has("title") ? data.getString("title") : "");
             buttonOk.setText(data.has("okButtonText") ? data.getString("okButtonText") : "Cancel");
             buttonCancel.setText(data.has("cancelButtonText") ? data.getString("cancelButtonText") : "Ok");
             description.setText(data.has("description") ? data.getString("description") : "");
             link = data.has("link") ? data.getString("link") : null;
             endPoint = data.has("endPoint") ? data.getString("endPoint") : null;
             method = data.has("method") ? data.getString("method") : null;
-            reqBody = data.has("reqBody") ? (JSONObject) data.get("reqBody") : null;
+            reqBody = data.has("reqBody") ?  data.getString("reqBody") : null;
             actions = data.has("actions") ? data.getJSONArray("actions") : null;
+            secondaryActions = data.has("secondaryActions") ? data.getJSONArray("secondaryActions") : null;
+            toastMessage = data.optString("toastMessage", null);
+            supportPhoneNumber = data.optString("contactSupportNumber", null);
```

Comment:
Here we are
Passing 'null' argument to parameter annotated as @NotNull 
Can you check this optString implementation, Looks like passing null as second parameter is not required